client/extensions/woocommerce/app/settings/email/email-settings/index.js

```diff
@@ -53,6 +53,14 @@ const originNotifications = [
 		subtitle: translate( 'If recipients reply to store emails they will be sent to this address.' ),
 		checkEmail: true,
 	},
+	{
+		field: 'email',
+		option: 'woocommerce_email_footer_text',
+		title: translate( 'Footer text' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **9** times:
`translate( 'Footer Text' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).